flask_server.py

```diff
@@ -33,6 +34,10 @@
 USERNAME = "testuser@gmail.com" # get_current_userkey()
 
 
+class UnauthorizedAccess(Exception):
```

Comment:
Oh I didn't see that those were from `cesium`; in that case it definitely shouldn't be in there, the library has no notion of authentication...